bcdaworker/main.go

```diff
@@ -181,36 +181,35 @@ func createDir(path string) error {
 	return nil
 }
 
-func writeBBDataToFile(ctx context.Context, bb client.APIClient, db *gorm.DB, acoID string, acoCMSID string, cclfBeneficiaryIDs []string, jobID, t, since string, transactionTime time.Time) (fileUUID string, error error) {
+func writeBBDataToFile(ctx context.Context, bb client.APIClient, db *gorm.DB, cmsID string, jobArgs models.JobEnqueueArgs) (fileUUID string, err error) {
 	segment := getSegment(ctx, "writeBBDataToFile")
 	defer func() {
 		if err := segment.End(); err != nil {
 			log.Error(err)
 		}
 	}()
 
-	if bb == nil {
```

Comment:
Since writeBBDataToFile is package private, we can guarantee that client.APIClient is properly initialized. As such, we do not need a nil check.